project/dependencies.scala

```diff
@@ -1,72 +1,124 @@
 import sbt.{Def, _}
 import sbt.Keys._
+import sbt.plugins.JvmPlugin
 
-object dependencies {
+object dependencies extends AutoPlugin {
 
   object V {
 
-    val cats                = "2.1.1"
-    val `cats-laws`         = "2.1.0"
-    val circe               = "0.13.0"
-    val `discipline-specs2` = "1.0.0"
-    val doobie              = "0.8.8"
-    val http4s              = "0.21.1"
-    val specs               = "4.8.3"
-    val shapeless           = "2.3.3"
-    val `par-colls`         = "0.2.0"
+    val `cats-effect` = on {
+      case (2, 12) => "[0.2,)"
+      case (2, 13) => "[2.0.0,)"
+    }
+
+    val circe = on {
+      case (2, 12) => "[0.6.0,)"
+      case (2, 13) => "[0.12.0,)"
+    }
+
+    val doobie = on {
+      case (2, 12) => "[0.4.0,)"
+      case (2, 13) => "[0.8.2,)"
+    }
+
+    val http4s = on {
+      case (2, 12) => "[0.15.0,)"
+      case (2, 13) => "[0.21.0,)"
+    }
+
+    val scalacheck = "[1.14.0,)"
```

Comment:
Didn't know about this 🚀

Reply:
I've just updated the PR with a reference to a [maven version range reference docs](https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN8903). Hope it's useful :)